### DISCUSSION TITLE
somebody for mac users saw a damaged error to delete the app

### DISCUSSION THREAD

**Discussion by otakususumu95** [stakira/OpenUtau#1199] [Q&A]
can you fix this problem?

**Comment by oxygen-dioxide** [comment]
Open a terminal and run `xattr -rc /Applications/OpenUtau.app`. Try opening OpenUtau again.
See https://github.com/stakira/OpenUtau/wiki/FAQ#on-macos-when-i-try-to-run-openutau-it-says-this-app-is-damaged
